app/frontend/src/cljs/teet/asset/cost_items_controller.cljs

```diff
@@ -518,15 +519,24 @@
                                          :total-cost
                                          :quantity-unit :type
                                          :ui/group)
-                     :price price}
-           :result-event ->SaveCostGroupPriceResponse}))
+                     ;; Interpret empty string as 0
+                     :price (if (str/blank? price) "0" price)}
```

Comment:
I don't understand why empty is zero?

There's a difference between something being free and the price not being known for it.
You should be able to retract a price without setting it to zero.

Reply:
Talked about this in Slack, let's change to retract.